### ISSUE TITLE
Can't call method "sas_time" on an undefined value

### ISSUE THREAD

**Issue by eserte** [whosgonna/Azure-SAS-Timestamp#1]
On some of my smoker systems the test suite fails:
```
#   Failed test 'Azure::SAS::Timestamp->new() died'
#   at t/110_values_timestamp.t line 17.
#     Error was:  Error parsing time at /usr/perl5.20.1Dp/lib/5.20.1/amd64-freebsd-ld/Time/Piece.pm line 469.
Can't call method "sas_time" on an undefined value at t/110_values_timestamp.t line 23.
# Tests were run but no plan was declared and done_testing() was not seen.
# Looks like your test exited with 255 just after 3.
t/110_values_timestamp.t .. 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 1/3 subtests 
```

Statistical analysis suggests that this happens if an older version of Time::Piece is installed (1.31 is not OK, 1.3204 is OK). If this is really the case, then a minimum prereq version should be specified.
```
****************************************************************
Regression 'mod:Time::Piece'
****************************************************************
Name           	       Theta	      StdErr	 T-stat
[0='const']    	      0.0000	      0.1203	   0.00
[1='eq_1.27']  	     -0.0000	      0.1701	  -0.00
[2='eq_1.29']  	      0.0000	      0.1473	   0.00
[3='eq_1.31']  	     -0.0000	      0.1276	  -0.00
[4='eq_1.3204']	      1.0000	      0.1345	   7.44
[5='eq_1.33']  	      1.0000	      0.1236	   8.09
[6='eq_1.34']  	      0.9545	      0.1230	   7.76
[7='eq_1.3401']	      1.0000	      0.1236	   8.09

R^2= 0.911, N= 74, K= 8
****************************************************************
```

**Comment by whosgonna** [comment]
Thank you.  it looks like the problem is with the use of `%F` in strptime which was fixed in 1.31_02.  Rather than setting a minimum version, I'll just change any usage of %F in the logic to `%Y-%m-%d` .

Reference Time::Piece issue [122092](https://rt.cpan.org/Public/Bug/Display.html?id=122092)

**Comment by whosgonna** [comment]
Should be fixed in v 0.0.3 just released to cpan.  Thank you!
